docs/rendering/modelextensions/rendertypes.md

```diff
@@ -0,0 +1,74 @@
+Render Types
+============
+
+Adding the `render_type` entry at the top level allows specifying the render type this model will render in.  
+If this is not specified, the model will fall back to the render types returned by `ItemBlockRenderTypes#getRenderLayers()`.
+
+!!! note
+    Since 1.19 this replaces the deprecated method of setting the applicable render type(s) via `ItemBlockRenderTypes#setRenderLayer()` for blocks.
+
+Example of a cutout block (i.e. a Glass Block)
+
+```js
+{
```

Comment:
I would probably add a comment here saying something like `// For some model JSON`